src/server/api/controllers/cvs.controller.js

```diff
@@ -4,18 +4,30 @@ const knex = require('../../config/db');
 const Error = require('../lib/utils/http-error');
 const moment = require('moment-timezone');
 
-const getCvs = async () => {
+const getCvs = async (title, limit) => {
   try {
-    return await knex('cv').select('cv.id', 'cv.title');
+    if (title) {
+      return await knex('cvs')
+        .select('cvs.id', 'cvs.title')
+        .where({ title });
+    } else if (limit) {
+      return await knex('cvs')
+        .select('cvs.id', 'cvs.title')
+        .limit({ limit });
+    } else
+      return await knex('cvs')
+        .select('cvs.id', 'cvs.title', 'cvs.file_url', 'cvs.createdAt')
```

Comment:
Cant you just say select? or maybe `select(*)`

```suggestion
        .select()
```